cluster.yml

```diff
@@ -13,6 +13,19 @@
   vars:
     ansible_connection: local
 
+- hosts: localhost
+  tasks:
+    - name: deploy warning for non kubeadm
+      debug:
+        msg: "non-kubeadm code deprecated in v2.9  (TODO remove in v3.0)"
```

Comment:
`DEPRECATION: non-kubeadm deployment is deprecated from v2.9. Will be removed in next release.`

Reply:
Thanks